pkgs/servers/jellyfin/update.sh

```diff
@@ -1,11 +1,8 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl jq common-updater-scripts dotnetCorePackages.sdk_6_0 nuget-to-nix gnused nix coreutils findutils
+#!nix-shell -i bash -p curl jq common-updater-scripts gnused nix coreutils
```

Comment:
This seems to be missing `-I nixpkgs=../../..`?
```suggestion
#!nix-shell -I nixpkgs=../../.. -i bash -p curl jq common-updater-scripts gnused nix coreutils
```